packages/yourloops/lib/auth/api.ts

```diff
@@ -152,35 +154,49 @@ class API extends EventTarget {
       },
     });
 
-    if (response.ok && response.status === http.StatusOK) {
-      this.sessionToken = response.headers.get(SESSION_TOKEN_HEADER);
-    } else {
-      /** @type{APIErrorResponse} */
-      const responseBody = await response.json() as APIErrorResponse;
-      reason = t(responseBody.reason);
-    }
+    if (!response.ok || response.status !== http.StatusOK) {
+
+      switch (response.status) {
+        case http.StatusUnauthorized:
+          // eslint-disable-next-line no-undefined
+          if (appConfig.MAX_FAILED_LOGIN_ATTEMPTS !== undefined) {
+            if (++this.wrongCredentialCount >= appConfig.MAX_FAILED_LOGIN_ATTEMPTS) {
+              reason = `Your account has been locked for ${appConfig.DELAY_BEFORE_NEXT_LOGIN_ATTEMPT} minutes. You have reached the maximum number of login attempts.`;
+            } else {
```

Comment:
this one may not work as expected. 
The counter is done at the application level while it should be done at the user level.
If you try to connect 4 times with 4 different wrong accounts, you will get the message saying that your account is locked, which is not true.

Reply:
correct , good catch

Reply:
https://diabeloop.atlassian.net/browse/YLP-454